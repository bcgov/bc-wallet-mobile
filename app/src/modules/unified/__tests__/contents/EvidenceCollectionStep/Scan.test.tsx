import { render } from '@testing-library/react-native'
import React from 'react'

import ScanContent from '../../../contents/EvidenceCollectionStep/Scan'

describe('ScanContent Component', () => {
  const goToBirthdate = jest.fn()

  beforeEach(() => {
    jest.resetAllMocks()
  })

  test('renders correctly', () => {
    const tree = render(<ScanContent goToBirthdate={goToBirthdate} />)
    expect(tree).toMatchSnapshot()
  })
})
