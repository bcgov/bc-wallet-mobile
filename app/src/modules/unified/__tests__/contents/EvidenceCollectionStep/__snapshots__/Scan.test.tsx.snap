// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScanContent Component renders correctly 1`] = `
<RNCSafeAreaView
  edges={
    [
      "bottom",
      "left",
      "right",
    ]
  }
  style={
    {
      "backgroundColor": "#313132",
      "height": "100%",
      "justifyContent": "space-between",
    }
  }
>
  <RCTScrollView
    contentContainerStyle={
      {
        "flex": 1,
        "padding": 24,
      }
    }
  >
    <View>
      <Text
        style={
          {
            "color": "#FFFFFF",
            "fontSize": 26,
            "fontWeight": "bold",
          }
        }
      >
        Coming soon!
      </Text>
    </View>
  </RCTScrollView>
</RNCSafeAreaView>
`;
